packages/instantsearch.js/src/connectors/index.ts

```diff
@@ -30,6 +30,7 @@ export { default as connectNumericMenu } from './numeric-menu/connectNumericMenu
 export { default as connectPagination } from './pagination/connectPagination';
 export { default as connectRange } from './range/connectRange';
 export { default as connectRefinementList } from './refinement-list/connectRefinementList';
+export { default as connectRelatedProducts } from './related-products/connectRelatedProducts';
```

Comment:
I see the reasons _for_ doing it like this, but we aren't releasing this as `EXPERIMENTAL_` then?

Reply:
Nope we're going GA directly.